cypress/commons/constants/brewery/IdConstants.js

```diff
@@ -12,6 +12,10 @@ export const BREWERY_SELECTORS = {
         waitersDisabledValue: '[data-cy=nb_waiters-value-disabled-input]',
         waitersInput: '[id=number-input-nb_waiters]',
       },
+      fileUpload: {
+        tabName: '[data-cy=file_upload_tab]',
+        initialStockDataset: '[data-cy=file-upload-initial_stock_dataset]',
```

Comment:
I dont't realy understand the typo convention. Why don't we write data-cy only with dash `-` ?

Reply:
I think it's because of the difference between our data-cy convention (dashes) and id naming convention (underscores), concatenating `data-cy` + `id` gives this thing :slightly_smiling_face: